Gemfile

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+
+gem 'kitchen-docker', '>= 2.9'
+gem 'kitchen-inspec', '>= 1.1'
+gem 'kitchen-salt', '>= 0.6.0'
+# Latest versions of `train` cause failure when running `kitchen verify`
+# Downgrading to `3.2.0` until this is fixed upstream
+# https://github.com/inspec/train/pull/544#issuecomment-566055052
+gem 'train', '3.2.0'
```

Comment:
No longer necessary since fixed upstream and we're also adding the `Gemfile.lock` now, will be resolved when commit is added here:

https://github.com/myii/jetbrains-pycharm-formula/commit/a3bac46edc0678d4436b425e3929d5c04aea72f6#diff-8b7db4d5cc4b8f6dc8feb7030baa2478L8-L11